src/Components/Web.JS/.gitignore

```diff
@@ -1,3 +1,4 @@
 ﻿node_modules/
-dist/
-dist/
+dist/Debug/
+dist/Release/blazor.webassembly.js
+dist/Release/blazor.webview.js
```

Comment:
If this is ignored, how do these files get updated by devs when they make changes.